frontend/src/utils/api/api.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+
+import axios from 'axios';
+import { APIEndpoints } from './types/endpoints';
+import composeEndpoint from './composeEndpoint';
+
+const API: APIEndpoints = {
```

Comment:
Do we need to define a type for this Object? It would be better if we can just infer the type of this object so that in future, we can easily add a new route without touching a type declaration